controllers/pagerdutyintegration/serviceorchestration_handler.go

```diff
@@ -100,13 +99,13 @@ func (r *PagerDutyIntegrationReconciler) handleServiceOrchestration(pdclient pd.
 	}
 
 	serviceOrchestrationConfigMap := types.NamespacedName{
-		Name:      pdi.Spec.ServiceOrchestration.RuleConfigConfigMapRef.Name,
+		Name:      orchestrationConfigmapName,
 		Namespace: pdi.Spec.ServiceOrchestration.RuleConfigConfigMapRef.Namespace,
 	}
 
-	ruleConfigmap := &corev1.ConfigMap{}
+	orchestrationRuleConfigmap := &corev1.ConfigMap{}
 
-	err = r.Get(context.TODO(), serviceOrchestrationConfigMap, ruleConfigmap)
+	err = r.Get(context.TODO(), serviceOrchestrationConfigMap, orchestrationRuleConfigmap)
```

Comment:
This feels redundant as the `utils.LoadConfigMapData` will be loading the configmap anyway, and reporting it if fails. It seems only needed for the sake of `GetLabels()` check, which as I've noted in a separate comment I'm not sure is all that essential.

Reply:
It is still needed? I think the `utils.LoadConfigMapData` func cannot handle the configmap absent situation.

Reply:
The first thing `utils.LoadConfigMapData` does is do a `Get` on the configmap and return an error if that fails, so it should handle it fine.

Reply:
Tested and confirmed it works as before after remove this. Thanks!